### PR TITLE
NS-46 Improve access to SIS section and enrollment data

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/NS-46

Another shuffle of the external-data deck.

### CODE REVIEW

**Comment by raydavis** [comment]
I'm closing this to check on a possibly missing piece...

**Comment by raydavis** [comment]
Phew. False alarm, due to the SIS API's many flavors of "displayName".

File: nessie/sql_templates/create_intermediate_schema.template.sql [hunk 1]
```diff
@@ -65,26 +111,20 @@ AS (
         s.name AS canvas_section_name,
         et.name AS canvas_course_term,
         extracted_section_ids.sis_term_id,
-        extracted_section_ids.sis_section_id,
-        sc.course_display_name AS sis_course_name,
-        sc.course_title AS sis_course_title,
-        sc.instruction_format AS sis_instruction_format,
-        sc.section_num AS sis_section_num,
-        sc.is_primary AS sis_primary
+        extracted_section_ids.sis_section_id
```
Comment (paulkerschen) [inline nessie/sql_templates/create_intermediate_schema.template.sql]: @raydavis Sorry for the drive-by - I think the bug you found in NS-46 isn't that Canvas-less SIS sections are missing, but that Canvas-less SIS sections have null `sis_term_id` and `sis_section_id`. Nessie SQL is pulling from the wrong side of the join and should (I think) be using `sc`, not `extracted_section_ids`. This is just a hunch; apologies if you're way ahead of me.

**Comment by raydavis** [comment]
Oh, good eye, @paulkerschen ! (Wish I could say as much for the clavicle.) I'll amend `course_sections` accordingly. But I think I'll leave `sis_enrollments` there rather than trying a similarly complex outer join in `active_student_enrollments` -- I'm in a maze of twisty little BOAC data structures, and scared to start again from the beginning.

**Comment by raydavis** [comment]
PR amended.

**Comment by raydavis** [comment]
Back to the kitchen again. I've been so focused on the analytics side of the house that I lost sight of the UX's dependency on section instructors-and-schedules.
